app/views/admin/services/_results.html.erb

```diff
@@ -11,7 +11,7 @@
     </tr>
   </thead>
   <tbody>
-    <% @services.each do |service| %>
+    <% services.each do |service| %>
```

Comment:
`@services` relies on the instance variable being available from the controller,
whereas `services relies on the variable being passed into the partial